### DISCUSSION TITLE
Migrating data to Saleor Cloud database

### DISCUSSION THREAD

**Discussion by joeydeckers** [saleor/saleor#11575] [Q&A] [answered]
Hey guys,

We are currently running our own Saleor instance in Digital Ocean apps for a client. We are experience alot of issues with stability and we get alot of 524 timeout errors. Users sometimes users cannot add products to their cart. We are considering migrating to Saleor Cloud. As you can imagine we need to use the current database because of alot of data. Is it possible to use the database from Digital Ocean apps. Or how can we import the data from the current database to the Saleor Cloud database?

Thanks!

**Comment by timuric** [comment, marked as answer]
Hi @joeydeckers 

You can provide database dump of Saleor data and after security review we can upload it to your instance. Please use chat button in your cloud instance to get technical support.

Here is the command for creating SQL dump:

```
pg_dump -n public --no-subscriptions --no-publications --no-comments --quote-all-identifiers --no-owner --no-acl --schema-only 'postgres://rolename@hostname/dbname' > project.schema.sql

pg_dump -n public --no-subscriptions --no-publications --no-comments --quote-all-identifiers --no-owner --no-acl --data-only 'postgres://rolename@hostname/dbname' > project.data.sql
```

Please note that if you have custom models or plugins it will not be possible to use cloud version. We also advise upgrading to the latest version of Saleor before starting migration.

**Reply by dev-tiger01** [reply]
Hi @timuric, 
same request here.. We'd like to test the sandbox with our current self-hosted data, already did the db dump. Unfortunately, we don't see any chat possibility in the chat cloud instance...everything is pointing to Typeform. 
How can we get in touch with the tech support? 

Until now, we've been trying to self-host saleor but it's too much hassle, so going PRO plan makes sense for us.
Would it be even possible to get PRO plan for one day trial?

Cheers
